cmd/cli/uninstall_mesh.go

```diff
@@ -171,6 +171,17 @@ func (d *uninstallMeshCmd) run() error {
 			}
 
 			if err == nil {
+				namespaces, _ := selectNamespacesMonitoredByMesh(m.name, d.clientSet)
```

Comment:
@mudit-01 Could you add a unit test for the namespace cleanup on mesh uninstall? Thanks!

Reply:
Sure @jaellio I will do that thanks for the review

Reply:
@mudit-01 we are pending a updated unit test here. 

https://github.com/openservicemesh/osm/blob/d2c37a6279dae347e65144b175fed94d661cb711/cmd/cli/uninstall_mesh_test.go#L631

You can check if the OSM related labels/annotations are removed from existing namespaces after uninstallation.

Reply:
@jaellio @allenlsy added test for checking labels/annotations.